src/index.ts

```diff
@@ -50,14 +92,29 @@ export default class Account {
     ethUtil.defineProperties(this, fields, data)
   }
 
+  /**
+   * Returns the RLP serialization of the account as a `Buffer`.
+   *
+   * @return {Buffer}
```

Comment:
If the method is annotated, the docs don't need to specify the type. Same comment for `isContract` and `setCode`.

Reply:
updated!